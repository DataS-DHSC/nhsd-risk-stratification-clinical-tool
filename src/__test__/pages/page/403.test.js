import React from 'react';
import { render } from '@testing-library/react';
import Error403 from '../../../pages/page/403';

const setup = () => render(<Error403 />);

describe('403 page', () => {
  it('should match snapshot', () => {
    const container = setup();
    expect(container.asFragment()).toMatchSnapshot();
  });
});
