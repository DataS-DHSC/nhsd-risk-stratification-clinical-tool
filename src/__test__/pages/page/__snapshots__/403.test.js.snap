// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`403 page should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="nhsuk-grid-row"
  >
    <div
      class="nhsuk-grid-column-two-thirds"
    >
      <h1
        class="nhsuk-label-wrapper"
      >
        <label
          class="nhsuk-label nhsuk-label--xl"
        >
          You've not yet been given access to the COVID-19 Clinical Risk Assessment Tool
        </label>
      </h1>
      <p>
        If you're a clinician and want to use the COVID-19 Clinical Risk Assessment Tool, 
        <a
          href="/login"
          id="login-link"
        >
           generate a link to sign in.
        </a>
      </p>
      <p>
        As a clinician, you can 
        <a
          href="https://digital.nhs.uk/coronavirus/risk-assessment/clinical-tool#access-the-tool"
          id="clinician-info-link"
          rel="noreferrer"
          target="_blank"
        >
          find out how to get access to the tool (opens in new window).
        </a>
      </p>
      <p>
        If you're not a clinician and you have a question about the tool, you should read the 
        <a
          href="https://digital.nhs.uk/coronavirus/risk-assessment/clinical-tool/information-for-patients"
          id="patient-guidance-link"
          rel="noreferrer"
          target="_blank"
        >
          Information for Patients (opens in new window)
        </a>
         or email risk.strat.spl@nhs.net
      </p>
    </div>
  </div>
</DocumentFragment>
`;
